setup.py

```diff
@@ -17,33 +12,31 @@
     "passlib>=1.6",
     "webtest",
 ]
-if sys.version_info == (2, 7):
-    tests_require.append("mock")
 
-setup_requires = ["setuptools", "setuptools-git >= 0.3"]
-if sys.version_info >= (3, 5):
-    setup_requires.append("wheel >= 0.25.0")  # earlier wheels fail in 3.5
-else:
-    setup_requires.append("wheel")
+setup_requires = ["setuptools", "setuptools-git >= 0.3", "wheel >= 0.25.0"]
+
+
+def read_file(rel_path: str):
+    return Path(__file__).parent.joinpath(rel_path).read_text()
 
 
 def get_version():
-    d = {}
+    locals_ = {}
     try:
-        do_exec(open("pypiserver/__init__.py").read(), d)  # @UndefinedVariable
+        exec(read_file("pypiserver/__init__.py"), locals_)
     except (ImportError, RuntimeError):
         pass
-    return d["__version__"]
+    return locals_["__version__"]
 
 
 setup(
     name="pypiserver",
     description="A minimal PyPI server for use with pip/easy_install.",
-    long_description=open("README.rst").read(),
+    long_description=read_file("README.rst"),
     version=get_version(),
     packages=["pypiserver"],
     package_data={"pypiserver": ["welcome.html"]},
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    python_requires=">=3.6",
```

Comment:
Good catch on this! Forgot to update the metadata when I dropped the last few versions.